packages/wonder-blocks-form/src/components/__tests__/labeled-text-field.test.js

```diff
@@ -485,3 +486,133 @@ describe("LabeledTextField", () => {
         expect(textField).toHaveProp("autoComplete", autoComplete);
     });
 });
+
+describe("Required LabeledTextField", () => {
+    test("Has * when required prop is true", async () => {
+        // Arrange
+        // Act
+        render(
+            <LabeledTextField
+                label="Label"
+                value=""
+                onChange={() => {}}
+                required={true}
+            />,
+        );
+
+        // Assert
+        expect(screen.getByText("*")).toBeInTheDocument();
+    });
+
+    test("Does not have * when required prop is false", () => {
+        // Arrange
+        // Act
+        render(
+            <LabeledTextField
+                label="Label"
+                value=""
+                onChange={() => {}}
+                required={false}
+            />,
+        );
+
+        // Assert
+        expect(screen.queryByText("*")).not.toBeInTheDocument();
+    });
+
+    test("aria-required is true when required prop is true", () => {
+        // Arrange
+        // Act
+        render(
+            <LabeledTextField
+                label="Label"
+                value=""
+                onChange={() => {}}
+                testId="foo-labeled-text-field"
+                required={true}
+            />,
+        );
+
+        const textField = screen.getByTestId("foo-labeled-text-field-field");
+
+        // Assert
+        expect(textField).toHaveAttribute("aria-required", "true");
+    });
+
+    test("aria-required is false when required prop is false", () => {
+        // Arrange
+        // Act
+        render(
+            <LabeledTextField
+                label="Label"
+                value=""
+                onChange={() => {}}
+                testId="foo-labeled-text-field"
+                required={false}
+            />,
+        );
+
+        const textField = screen.getByTestId("foo-labeled-text-field-field");
+
+        // Assert
+        expect(textField).not.toHaveAttribute("aria-required", "true");
+    });
+
+    /*
+    test.only("displays the default message when requiredErrorMessage is not passed in", async () => {
+        // Arrange
+        render(
+            <LabeledTextField
+                label="Label"
+                value=""
+                onChange={() => {}}
+                required={true}
+                testId="test-labeled-text-field"
+            />,
+        );
+
+        const textField = screen.getByTestId("test-labeled-text-field-field");
+        textField.focus();
+        userEvent.type(textField, "a");
+
+        await wait(0);
+
+        console.log("value", textField.value);
```

Comment:
(WIP comment):
I can't seem to get the field to change its value in the test.

It looks like the stories in storybook have to use some sort of helper to get the value to change on keyboard input?

https://github.com/Khan/wonder-blocks/blob/master/packages/wonder-blocks-form/src/components/labeled-text-field.stories.js#L20